modules/augment/include/opencv2/augment/rotate.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_ROTATE_HPP
+#define OPENCV_AUGMENT_ROTATE_HPP
+#include <opencv2/augment/Transform.hpp>
+
+namespace cv {
+namespace augment {
+
+    //! @addtogroup augment
+        //! @{
+
+    class CV_EXPORTS_W Rotate : public Transform
```

Comment:
Let's focus on more important code. I'd like to propose you to format it in this way:

```cpp
namespace cv { namespace augment {

class CV_EXPORTS_W Rotate : public Transform
{
    // ...
};

}}  // namespace cv::augment
```

Reply:
Ok, I will do that

Reply:
done